app/src/androidTest/java/com/example/concordia_campus_guide/InstrumentalTests/LocationFragmentInstrumentalTest.java

```diff
@@ -46,5 +68,38 @@ public void styleMarker(){
         assertNotNull(bitmap2);
     }
 
+//    @Test
```

Comment:
Same as before, can we add a todo and create an issue to fix the test?

Reply:
Having this much commented out code will increase our code smells... Is there a way to avoid this Hampic ?

Reply:
For particular tests, we can put the @Ignore annotation in front or delete the @test annotation. Can we do that instead if commenting out things?

Reply:
I will put the ignore. and put a todo on it because it is a test we can fix later. Good catch 👍

Reply:
Actually, instead of Todo, I will create a task issue for it.